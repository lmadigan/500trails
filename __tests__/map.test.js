import React from 'react';
import renderer from 'react-test-renderer';
import Session from '../frontend/components/session/session_form.jsx';

describe('Google Map', () => {
  it('Creates a map', () => {
    const map = renderer.create(<Session />);
    expect(map).toMatchSnapshot();
  });
});
