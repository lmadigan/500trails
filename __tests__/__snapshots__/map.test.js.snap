// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Google Map Creates a map 1`] = `
<div
  className="loggin-form-page"
>
  <section
    className="login-form-wrapper"
  >
    <section
      className="login-error-bar"
    />
    <div
      className="inner-form"
    >
      <form
        className="login-form"
        onSubmit={[Function]}
      >
        <div
          className="login-content"
        >
          <h1
            className="login-header"
          >
            Join 500trails
          </h1>
        </div>
        <div
          className="field-item"
        >
          <input
            className="login-input"
            onChange={[Function]}
            placeholder="Username"
            type="text"
            value=""
          />
        </div>
        <br />
        <div
          className="field-item"
        >
          <br />
          <input
            className="login-input"
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value=""
          />
        </div>
        <br />
        <div
          className="field-item"
        >
          <input
            className="submitButton"
            type="submit"
            value={undefined}
          />
        </div>
        <div
          className="switch-message"
        />
      </form>
    </div>
  </section>
</div>
`;
